### PR TITLE
[YUNIKORN-2211] Replace Allocation uuid with allocationID

### PR DESCRIPTION
**Pull request by manirajv06**
### What is this PR for?
docs changes to replace uuid with allocationID   

### What type of PR is it?
* [ ] - Documentation

### Todos
* [ ] - Task

### What is the Jira issue?
https://issues.apache.org/jira/browse/YUNIKORN-2211

### Questions:
* [ ] - The licenses files need update.
* [ ] - There is breaking changes for older versions.
* [ ] - It needs documentation.

### CODE REVIEW

**Comment by manirajv06** [comment]
@wilfred-s can you take a look?

File: docs/api/scheduler.md [hunk 1]
```diff
@@ -285,6 +285,10 @@ Fetch all Applications for the given Partition/Queue combination and displays ge
 
 **Code** : `200 OK`
 
+**Deprecated**:
+
+Field `uuid` has been deprecated, would be removed from below response in YUNIKORN 1.7.0 release. `AllocationID` has replaced `uuid`. Both `uuid` and `AllocationID` fields has same value. `AllocationID` has extra suffix containing hypen and counter (-0, -1 and so on..) at the end.
```
Comment (pbacsko) [inline docs/api/scheduler.md:290]: "have the same value"
"and so on" (no "..")
"hyphen"

File: docs/api/scheduler.md [hunk 2]
```diff
@@ -540,6 +547,10 @@ Fetch an Application given a Partition, Queue and Application ID and displays ge
...
```
Comment (pbacsko) [inline docs/api/scheduler.md:552]: "have the same value"
"and so on" (no "..")
"hyphen"

**Review by pbacsko — CHANGES_REQUESTED** [review]
Some language related comments

**Review by pbacsko — APPROVED** [review]
+1

**Pull request closed without merging**